wordpress-mobile/WordPress-iOS
# Removes the question mark button from the post editor.

Removes the question mark button from the post editor.

![simulator screen shot - iphone 8 - 2018-03-05 at 17 40 18](https://user-images.githubusercontent.com/1836005/36998625-5353c0a6-209c-11e8-8565-fd0fdcb370e4.png)

**How to test:**

In both iPhone and iPad, create a new post and play with the title text field.
Rotate the device.

Make sure the title dimensions and position is always as expected.
